astroplan/tests/test_core.py

```diff
@@ -806,6 +1090,6 @@ def test_Observer_init(self):
             Observer(location=EarthLocation(0, 0, 0), timezone=-6)
 
     def test_Observer_altaz(self):
-        with self.assertRaises(TypeError):
+        with self.assertRaises(ValueError):
```

Comment:
I don't like `unittest.TestCase` ... `pytest` is much nicer.
Let's see if others agree first before you change this.